src/examples/canvas/test_shader.jsx

```diff
@@ -60,6 +61,7 @@ const TestShader = (props) => {
       {...props}
     >
       <boxBufferGeometry args={[0.5, 0.5, 0.5]} />
+      {/* @ts-ignore */}
```

Comment:
Would it be better to extend the type definition of `@react-three/fiber`?

Reply:
@shuta13 We don't need to spend time on that example as it's not the core. @ts-ignore is fine enough for the examples folder 👍